cmd/debos/debos.go

```diff
@@ -117,10 +117,10 @@ func DumpActionFields(iface interface{}, depth int) {
 			case reflect.Slice:
 				s := reflect.ValueOf(f.Interface())
 				if s.Len() > 0 && s.Index(0).Kind() == reflect.Struct {
-					log.Printf("    %s:\n", entries.Type().Field(i).Name)
+					log.Printf("%s  %s:\n", tab, entries.Type().Field(i).Name)
```

Comment:
Your commit doesn't explain what this does; Should go into a seperaet MR as well maybe?